### PR TITLE
revision-3

### PR DESCRIPTION
**Pull request by lionel-rigoux**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/lionel-rigoux/test-revision/pull/4" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 Pretty bla bla bla. Prettier bla bla.
 
 This is a better new paragraph.
+
+A new paragraph.
```
Comment (lionel-rigoux) [inline README.md]: comments
!fix

Comment (lionel-rigoux) [inline README.md]: !fixed

File: README.md [hunk 2]
```diff
@@ -3,3 +3,5 @@
 Pretty bla bla bla. Prettier bla bla.
 
 This is a better new paragraph.
+
```
Comment (lionel-rigoux) [inline README.md]: we should add this

**Pull request merged**